### PR TITLE
Sync keywords among repos

### PR DESCRIPTION
**Pull request by jobselko**
This PR moves and updates NVD keywords to keep all three repos consistent (Assembler, OSIDB, guitoutils).

Related to OSIDB-2107

### CODE REVIEW

**Comment by jobselko** [comment]
FYI, nothing was removed, only alphabetically sorted.

**Review by osoukup — CHANGES_REQUESTED** [review]

File: collectors/nvd/keywords.py [hunk 1]
```diff
@@ -285,205 +421,355 @@
...
+    r"plugin <= [0-9\.]+ at WordPress",
+    r"plugins.*wordpress",
+    r"Secomea (GateManager|SiteManager)",
+    r"SIMATIC.*(PCS|CP)",
+    r"siteserver (CMS|SSCMS)",
+    r"Tenda( |_.*)",
+    r"TP-Link.*(TL|AX10v1|Tapo)",
+    r"VIDEOJET.*[\n]*.*psirt",
+    r"win32k\.sys",
+    r"WordPress.*plugin",
```
[the remark this answers is not in the document]
Comment (jobselko) [reply collectors/nvd/keywords.py:772]: This list is the exact copy of `guitoutils` and `Assembler` (see links in Jira). My aim is to keep it consistent everywhere to be able to find out what has changed quickly.

I could change it to sth like `[..., 'WPS Office', r'WordPress.*plugin', 'Zikula Application Framework', ...]`, but then I would have to do it everywhere.

[the remark this answers is not in the document]
Comment (osoukup) [reply collectors/nvd/keywords.py:772]: I have exactly the same aim. When I am looking for what has changed, I proceed alphabetically. Just try this
```
sorted(["def", r"abc"])
```

[the remark this answers is not in the document]
Comment (jobselko) [reply collectors/nvd/keywords.py:772]: Yeah, I am just trying that :). The output is ['abc', 'def']. Note that **r** is removed.

[the remark this answers is not in the document]
Comment (jobselko) [reply collectors/nvd/keywords.py:772]: But adding **r** to 30 items should not be a problem.

[the remark this answers is not in the document]
Comment (jobselko) [reply collectors/nvd/keywords.py:772]: Is it your last word that I should do it? :)

[the remark this answers is not in the document]
Comment (osoukup) [reply collectors/nvd/keywords.py:772]: My word is that the list should be idempotent against Python sorting unless you convince me that there is valid and good reason not to it this way.

[the remark this answers is not in the document]
Comment (jobselko) [reply collectors/nvd/keywords.py:772]: Agree with you, I will do it.

**Review by osoukup — APPROVED** [review]
LGTM

**Pull request merged**